lib/housekeeping.js

```diff
@@ -36,11 +36,29 @@ class HouseKeeper {
     this.maxInstanceLifeHours = maxInstanceLifeHours;
     this.monitor = monitor;
     this.tagger = tagger;
+
+    this.describeVolumesParams =  {
+      Filters: [{
+        Name: 'status',
+        Values: ['available', 'in-use'],
+      }],
+    } 
+  }
+
+  async handleVolumeData(volume) {}
```

Comment:
Javascript doesn't really have a built in concept about public/private for object properties.  Because of this, there's a weak convention of using an underscore as the first character of a purely internal property as a hint to the user that such properties shouldn't be used.  I think we should consider using that convention here for those functions which we know are internal to this class.

Reply:
That seems like a very good idea, thanks John!